### DISCUSSION TITLE
connecting older client versions to latest server

### DISCUSSION THREAD

**Discussion by sfrankel** [jamulussoftware/jamulus#3201] [General]
Hello -- are there any known issues connecting older client versions of Jamulus, eg 3.8.x, to the latest 3.10 release server?

My private server, running Jamulus v3.9.0 on Ubuntu 20.04, is having some issues and I'm considering upgrading to v3.10. But my bandmates are on ancient hardware that only supports older versions of Jamulus. Is that asking for trouble?

TIA!

**Comment by ann0see** [comment]
There shouldn't be problems if you update the server. You can always just install the old version again if it doesn't work.

**Reply by sfrankel** [reply]
OK sounds good. Thanks!
